### ISSUE TITLE
 DATE FORMAT

### ISSUE THREAD

**Issue by wpavanatti** [pluginsGLPI/formcreator#1050]
**Describe the bug**
I create a form, with one of the data type questions, in the answer, a data comes in the American format.

**To Reproduce**
1. Create a form with a question in type "Date"
2. Select as destination "tickets"
3. Submit Form
4. Go to "Accompaniments". See that the date format is in the American format "Y-M-D"

**Expected behavior**
I expected the return of the date in the Brazilian format "D-M-Y"

**Additional context**
Up to version 2.5.2, this problem did not exist From 2.6 in front the problem started.

**Comment by btry** [comment]
Hi

I began to investigate the issue. I don't know yet how to fix that. The challenge is that when the plugin generates the date for the ticket description it does not knows who will read the ticket, then I cannot choose his preferences. 

I need to check how this was handled in an older version of the plugin. I expect the format was set from requester's settings. 

In a multilingual context, and assuming that several users may read a ticket, the settings may match the preferences of a technician, but not an other technician. This is probably true in Formcreator 2.5.x also (again I need to check the code). 

In short: a date must be dynamically displayed depending on the preferences of the user when viewing the data, not when storing it in the DB. I'm afraid there is no perfect solution. If you have a better proposal, feel free to share it.

**Comment by btry** [comment]
Hi,

please test this above fix. It works for me.
